article/_posts/2020-10-05-Diva.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: review
+title: "Diva: Domain invariant variational autoencoders"
+tags: VAE adversarial domain-adaptation
+author: "Antoine Théberge"
+cite:
+    authors: "Maximilian Ilse, Jakub M. Tomczak, Christos Louizos, Max Welling"
+    title:   "Diva: Domain invariant variational autoencoders"
+    venue:   "MIDL 2020"
+pdf: "https://arxiv.org/pdf/1905.10427.pdf"
+---
+
+
+![](/article/images/diva/sc01.jpg)
+
+
+# Highlights
+
+
+
+
+The authors are working on data that has **different domain** and **different classes** and their goal is to disentangle the domain representation from the classes representation.  To do so, they built upon the paper [Learning Latent Subspaces in Variational Autoencoders](https://vitalab.github.io/article/2020/02/13/CSVAE.html).
+
+# Methods
+
+
+To do so, the authors learn 3 latent spaces (as opposed to 1 for a normal VAE) : 
+
+* $$z_d$$ the *domain* latent space;
+* $$z_x$$ the *data* latent space;
```

Comment:
I'd be a bit hesitant to call the z_x space the _data_ space, since its goal is to represent whatever variation cannot be explained through domain or class. So maybe something like the auxiliary space?